### DISCUSSION TITLE
Using same Dto name in application layer

### DISCUSSION THREAD

**Discussion by mehdihadeli** [kgrzybek/modular-monolith-with-ddd#209] [Ideas] [answered]
Hi,
In your code base you used same dto name like `UserDto` in [here](https://github.com/kgrzybek/modular-monolith-with-ddd/blob/37de06228f3f6f49de20a510d5349c784903b01f/src/Modules/UserAccess/Application/Authentication/Authenticate/UserDto.cs) and [here](https://github.com/kgrzybek/modular-monolith-with-ddd/blob/37de06228f3f6f49de20a510d5349c784903b01f/src/Modules/UserAccess/Application/Users/GetUser/UserDto.cs). Is it ok using same dto name with different name space? Is not better instead of same dto name we use something like `GetUserResponse` and `AuthenticatedUserResponse` for these cases?

**Comment by kgrzybek** [comment, marked as answer]
It is not a big deal but from the readability and clarity perspective, it should be cleaned up. So one of them should be renamed.

**Reply by mehdihadeli** [reply]
@kgrzybek Thanks
